x11-misc/bbtime/files/bbtime-0.1.5-asneeded.patch

```diff
@@ -2,8 +2,8 @@ Fixing build with as-needed
 
 https://bugs.gentoo.org/show_bug.cgi?id=248555
 
---- configure.in
-+++ configure.in
+--- a/configure.in
++++ b/configure.in
```

Comment:
`configure.in`... this sounds like `eautoreconf` (specifically, `autoconf`) complains about the "old style" input file, hence, after the `default` in `src_prepare`, do
`mv configure.{in,ac} || die`